about, experience, connect sections added

katy1313 opened the pull request.

Lee-Rice commented:
Hi Katya,

The HTML looks perfect to me. The automated checker did point out a couple typos. It's probably best to fix those before merging this. It's just your name in the first paragraph and the word engineer in two places under experience.

It would also be best to add a direct link to your GitHub like you did with your LinkedIn. Also, and this may have been intentional, but, just in case, the double quotations aren't needed around the text for LinkedIn. I'm just going to ask that these small issues be corrected. Resubmit and I'll go ahead and approve it for merge.

katy1313 commented:
> Hi Katya,
> 
> The HTML looks perfect to me. The automated checker did point out a couple typos. It's probably best to fix those before merging this. It's just your name in the first paragraph and the word engineer in two places under experience.
> 
> It would also be best to add a direct link to your GitHub like you did with your LinkedIn. Also, and this may have been intentional, but, just in case, the double quotations aren't needed around the text for LinkedIn. I'm just going to ask that these small issues be corrected. Resubmit and I'll go ahead and approve it for merge.

Hello Lee, I've addressed the issues mentioned above. Thanks!

Lee-Rice commented:
Thank you for taking care of those items. It looks like the first `<a href>` tag isn't closed.  That's causing the next line, the LinkedIn line, to link to GitHub and not LinkedIn. Would you mind fixing it and resubmitting?

My recommendation, and you may already be doing this, is to open the HTML file in a web browser while editing. That will allow you refresh the browser and see how the changes you've made look.

katy1313 commented:
> Thank you for taking care of those items. It looks like the first `<a href>` tag isn't closed. That's causing the next line, the LinkedIn line, to link to GitHub and not LinkedIn. Would you mind fixing it and resubmitting?
> 
> My recommendation, and you may already be doing this, is to open the HTML file in a web browser while editing. That will allow you refresh the browser and see how the changes you've made look.



> Thank you for taking care of those items. It looks like the first `<a href>` tag isn't closed. That's causing the next line, the LinkedIn line, to link to GitHub and not LinkedIn. Would you mind fixing it and resubmitting?
> 
> My recommendation, and you may already be doing this, is to open the HTML file in a web browser while editing. That will allow you refresh the browser and see how the changes you've made look.

All done! Do I need to resubmit my homework?

Lee-Rice commented:
> All done! Do I need to resubmit my homework?
Yep. Please resubmit it through the normal link. Looks good to me.

Lee-Rice commented:
Nice job. Looks great. Approved for merge.

katy1313 commented:
> Nice job. Looks great. Approved for merge.

Great, thanks! Should I delete the old branches?

Lee-Rice commented:
Weird. I replied to the email from GitHub and it didn't add the message here.

Feel free to delete the old branches once they're merged into main. If you ever need to go back to a previous version of your code, you can just clone from that point on the main branch.
